subscriptions/tests.py

```diff
@@ -546,93 +546,99 @@ def test_send_message_task_to_mother_text(self):
-                "delivered": False,
-                "attempts": 0,
-                "metadata": {},
-                "created_at": "2016-03-24T13:43:43.614952Z",
-                "updated_at": "2016-03-24T13:43:43.614921Z",
-            },
-            status=200,
-            content_type="application/json",
-        )
-
         # make messages
         message_data_eng_1 = {
             "messageset": existing.messageset,
             "sequence_number": 1,
             "lang": "eng_ZA",
+            "metadata": {"data": "message_data"},
             "text_content": "This is message 1",
         }
         Message.objects.create(**message_data_eng_1)
         message_data_eng_2 = {
             "messageset": existing.messageset,
             "sequence_number": 2,
             "lang": "eng_ZA",
+            "metadata": {"data": "message_data"},
             "text_content": "This is message 2",
         }
         Message.objects.create(**message_data_eng_2)
         message_data_eng_3 = {
             "messageset": existing.messageset,
             "sequence_number": 3,
             "lang": "eng_ZA",
+            "metadata": {"data": "message_data"},
             "text_content": "This is message 3",
         }
         Message.objects.create(**message_data_eng_3)
         message_data_zul_1 = {
             "messageset": existing.messageset,
             "sequence_number": 1,
             "lang": "zu_ZA",
+            "metadata": {"data": "message_data"},
             "text_content": "Ke msg 1",
         }
         Message.objects.create(**message_data_zul_1)
         message_data_zul_2 = {
             "messageset": existing.messageset,
             "sequence_number": 2,
             "lang": "zu_ZA",
+            "metadata": {"data": "message_data"},
             "text_content": "Ke msg 2",
         }
         Message.objects.create(**message_data_zul_2)
         message_data_zul_3 = {
             "messageset": existing.messageset,
             "sequence_number": 3,
             "lang": "zu_ZA",
+            "metadata": {"data": "message_data"},
             "text_content": "Ke msg 3",
         }
         Message.objects.create(**message_data_zul_3)
 
+        # mock identity address lookup
+        responses.add(
+            responses.GET,
+            (
+                "http://seed-identity-store/api/v1/identities/%s/addresses/msisdn"
+                "?default=True&use_communicate_through=True"
+            )
+            % (existing.identity,),
+            json={
+                "next": None,
+                "previous": None,
+                "results": [{"address": "+2345059992222"}],
+            },
+            status=200,
+            content_type="application/json",
+            match_querystring=True,
+        )
+
+        # Create message sender call
+        responses.add(
+            responses.POST,
+            "http://seed-message-sender/api/v1/outbound/",
+            json={
```

Comment:
This doesn't assert that the HTTP request contained these details, it's just what the mock will respond with when we make the HTTP request. We'll still need an assert to ensure that the request contains the metadata as we expect